content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,26 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, the container runtime and its underlying OCI runtime must support
+user namespaces. The following OCI runtimes offer support:
+
+* [crun][crun] version 1.9 or greater (we recommend version 1.13+).
```

Comment:
- (nit) We [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we
- (nit) It's easier for localizations to put the hyperlink URL inline

Reply:
Fixed, thanks